### ISSUE TITLE
False positive NestedFunctionCalls report when interpolating strings

### ISSUE THREAD

**Issue by krns** [rrrene/credo#1055]
### Environment

* Credo version (`mix credo -v`): 1.7.0
* Erlang/Elixir version (`elixir -v`): 1.14.3
* Operating system: Linux

### What were you trying to do?

```elixir
  defp translate(a), do: a

  defp test() do
    "#{translate("1")} #{translate("2")}"
  end
```

### Expected outcome

No warning

### Actual outcome

```
┃ [R] → Use a pipeline when there are nested function calls
┃       lib/my_application.ex:27 #(Module.test)
┃ [R] → Use a pipeline when there are nested function calls
┃       lib/my_application.ex:27 #(Module.test)
```

**Comment by randycoulman** [comment]
Duplicate of #1034?

**Comment by rrrene** [comment]
Yeah, closing this as a duplicate.

btw:  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```
